crates/matrix-sdk-sled/src/cryptostore.rs

```diff
@@ -217,7 +217,7 @@ impl SledStore {
             .temporary(false)
             .path(&path)
             .open()
-            .map_err(|e| CryptoStoreError::Backend(anyhow!(e)))?;
+            .map_err(|e| CryptoStoreError::Backend(anyhow!(e).into()))?;
```

Comment:
Can you try whether this works instead?

```suggestion
            .map_err(|e| CryptoStoreError::Backend(e.into()))?;
```

I don't think `anyhow` should be needed at all.

Reply:
CryptoStoreError is a different type, but I will add the changes to that one as well